.vale/styles/config/vocabularies/docs/accept.txt

```diff
@@ -66,6 +66,7 @@ subnet
 Trivy
 TLS
 templated
+underlaying
```

Comment:
That's not an English word – I guess `underlying` was meant to be used, which the linter should have in its dictionary?

Reply:
That is funny because the VScode plugin correct to this 😆